### ISSUE TITLE
Create boilerplate API project

### ISSUE THREAD

**Issue by dancingsushii** [amosproj/amos2024ss04-building-information-enhancer#38]
## User story
1. As a Backend Developer, I want to create a boilerplate for our geospatial data API, so that the team can have a consistent and standardized starting point for developing API endpoints.
2. I need a foundational project setup that incorporates our chosen technologies and best practices.
3. So that the team can accelerate development with a well-defined, scalable, and maintainable codebase.

## Acceptance criteria
* The project uses C# and the ASP.NET Core framework, set up with the latest stable version.
* A clear directory structure is established, following industry standards for API projects.
* The project includes a basic health check endpoint to validate the system status.
* The boilerplate includes a README file with setup instructions and a description of the project structure.

## Current state
![Image](https://github.com/amosproj/amos2024ss04-building-information-enhancer/assets/46463790/2f0e2bbf-734a-40c2-9cda-63f5dca53f7c)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
